content/kit/_index.md

```diff
@@ -28,19 +28,24 @@ In addition to the boards, your kit will also contain:
 
 | Part                         | Quantity | Specification               | Notes                                  |
 |------------------------------|----------|-----------------------------|----------------------------------------|
+| Battery Bag                  | 1        |                             |                                        |
 | Battery Charger              | 1        | iMAX B6                     |                                        |
 | Battery Charger Power Supply | 1        | 12V 5A                      |                                        |
 | Battery                      | 1        | 11.1V 2.2Ah Lithium Polymer |                                        |
-| USB Memory Stick             | 1        | 1GB                         |                                        |
+| Webcam                       | 1        | TeckNet C016                |                                        |
 | USB Hub                      | 1        |                             |                                        |
 | Micro USB Cable              | 2        |                             |                                        |
 | USB B Cable                  | 1        |                             |                                        |
-| Black Wire                   | 3m       |                             | Used for ground *only*                 |
-| Red Wire                     | 3m       |                             |                                        |
-| Blue Wire                    | 3m       |                             |                                        |
-| Small CamCon                 | 2        | 5mm                         |                                        |
 | CamCon                       | 2        | 7.5mm                       |                                        |
+| Small CamCon                 | 2        | 5mm                         |                                        |
 | Micro CamCon                 | 3        | 3mm                         | 1 additionally found on Pi power cable |
-| Webcam                       | 1        |                             |                                        |
-| Battery Bag                  | 1        |                             |                                        |
 | Pi Power Cable               | 1        | Micro USB to Micro CamCon   |                                        |
+
+### Parts we don't mind you keeping
+
+| Part                         | Quantity | Specification               | Notes                                  |
+|------------------------------|----------|-----------------------------|----------------------------------------|
+| Black Wire                   | 3m       |                             | Used for ground *only*                 |
+| Red Wire                     | 3m       |                             |                                        |
```

Comment:
Maybe add to the notes for red wire that it's for power only? :)

Reply:
That doesnt need to be done in the kit, it's in the wiring guide

Reply:
I'd go as far as to say that we don't need to say "black wire for ground only" on this page either. This page is just to list what kit there is and which parts we want back - it doesn't need to discuss the applications of each part.